test/scripts/cluster.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)
+
+function create_kind_cluster {
+  cluster=$1
+  podSubnet=$2
+  svcSubnet=$3
+
+  echo "Creating cluster '$cluster' in podSubnet='$podSubnet' and svcSubnet='$svcSubnet'"
+
+  kind create cluster --name "$cluster" --config=<<EOF
+apiVersion: kind.x-k8s.io/v1alpha4
+kind: Cluster
+networking:
+  podSubnet: "$podSubnet"
+  serviceSubnet: "$svcSubnet"
+EOF
+}
+
+function provision_kind_clusters {
+  echo "Creating KinD clusters"
+  kind_pids=()
+  create_kind_cluster east 10.10.0.0/16 10.255.10.0/24 &
+  kind_pids[0]=$!
+  create_kind_cluster west 10.30.0.0/16 10.255.30.0/24 &
```

Comment:
I was checking your PR locally and I noticed that these subnets are ignored, i.e. clusters use default subnets - 10.244.0.0/16 for pods and 10.96.0.0/16 for services, but they are also ignored on master, so it's not your fault. Instead of fixing it, we can remove these subnets, because mesh federation doesn't require non-overlapping CIDRs, so these custom subnets were always irrelevant.

Reply:
@jewertow I have a follow-up PR that further expands cluster-related functions for multiple peer setups.  I think this fix is a perfect addition to that one. WDYT?